styles.css

```diff
@@ -13,11 +13,23 @@
     width: 1em;
     height: 1em;
     vertical-align: middle;
-    margin-left: 0.3em;
+    margin-left: 0.5em;
     cursor: pointer;
 }
 
-.internal-link.internal-link-short-mode {
+/* Urgency score */
+.tasks-score {

```

Comment:
I've just noticed the difference between the comment and the name `.tasks-score`.

If I were looking to style the urgency, I would look for `urgency` in the name - plus, who knows, maybe there might be some other kind of score for tasks in the future?

So I think one of these is probably a better name for the addition:

- `.tasks-urgency`
- `.tasks-urgency-score`

Also, that would make it more consistent with the name of the instruction to show/hide it.

Reply:
I tried to avoid the term "urgency" because of [these thoughts](https://github.com/obsidian-tasks-group/obsidian-tasks/discussions/1157), but I agree that should be a separate discussion which I have started there.

For the time being, we should stick to the term "urgency" to be consistent, so I changed this.

Reply:
It has been renamed to `.tasks-urgency`